src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -461,6 +461,9 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="expandToMatching" value="Ctrl+Shift+E" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="expandToMatching" value="Ctrl+Shift+Alt+E" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="splitIntoLines" value="Cmd+Alt+A"/>
+         <shortcut refid="showDocumentOutline" value="Ctrl+Alt+G"/>
```

Comment:
That's right; just the name is poorly chosen. I also agree with stealing that keyboard shortcut.